controllers/storageoscluster/operator.go

```diff
@@ -60,14 +61,14 @@ func NewOperator(mgr ctrl.Manager, appsGetter tappv1.AppsV1Interface, kubeVersio
 	//              │                └──────────────┘
 	//              ▼
 	//          ┌────────┐
-	//    ┌─────┤  node  ├──┐──────────────────┐──────────────────┐
-	//    │     └────────┘  │                  │                  │
-	//    │                 │                  │                  │
-	//    │                 │                  │                  │
-	//    ▼                 ▼                  ▼                  ▼
-	// ┌─────┐       ┌─────────────┐   ┌────────────────┐   ┌──────────────┐
-	// │ csi │       │ api-manager │   │ portal-manager │   │ node-manager │
-	// └──┬──┘       └─────────┬───┘   └────────────────┘   └──────────────┘
+	//    ┌─────┤  node  ├──┐──────────────────┐──────────────────┐─────────────────────┐
+	//    │     └────────┘  │                  │                  │				   	    │
+	//    │                 │                  │                  │					    │
+	//    │                 │                  │                  │					    │
+	//    ▼                 ▼                  ▼                  ▼ 				    ▼
```

Comment:
slight formatting error here with the diagram

Reply:
It's just a tabs vs spaces thing, I changed to spaces